cicecore/cicedyn/analysis/ice_diagnostics.F90

```diff
@@ -420,21 +412,22 @@ subroutine runtime_diags (dt)
          do iblk = 1, nblocks
             do j = 1, ny_block
             do i = 1, nx_block
-               work1(i,j,iblk) = max(sqrt(uvelE(i,j,iblk)**2 &
-                                        + vvelE(i,j,iblk)**2), &
-                                     sqrt(uvelN(i,j,iblk)**2 &
-                                        + vvelN(i,j,iblk)**2))
+               work1(i,j,iblk) = max(sqrt(uvelE(i,j,iblk)**2 + vvelE(i,j,iblk)**2), &
+                                     sqrt(uvelN(i,j,iblk)**2 + vvelN(i,j,iblk)**2))
             enddo
             enddo
          enddo
          !$OMP END PARALLEL DO
       elseif (grid_ice == 'C') then
+         ! map uvelE to N and vvelN to E then compute max on E and N
+         call grid_average_X2Y('A',uvelE,'E',work1,'N')  ! work1 =~ uvelN
+         call grid_average_X2Y('A',vvelN,'N',work2,'E')  ! work2 =~ vvelE
          !$OMP PARALLEL DO PRIVATE(iblk,i,j)
          do iblk = 1, nblocks
             do j = 1, ny_block
             do i = 1, nx_block
-               work1(i,j,iblk) = sqrt(uvelE(i,j,iblk)**2 &
-                                    + vvelN(i,j,iblk)**2)
+               work1(i,j,iblk) = max(sqrt(uvelE(i,j,iblk)**2 + work2(i,j,iblk)**2), &
```

Comment:
kinetic energy is still BFB in the log files?

Reply:
I am comparing to the current CICE Consortium main.  So we expect/hope that will be bit-for-bit.  If I compared against the most recent version of E3SM CICE, then I expect the log files would be different.

Reply:
okay, that makes sense